runtime/jcl/common/shared.c

```diff
@@ -1933,6 +1933,104 @@ Java_com_ibm_oti_shared_SharedClassURLClasspathHelperImpl_notifyClasspathChange2
 #endif		/* J9VM_OPT_SHARED_CLASSES */
 }
 
+/* Notify the open state to all the jar/zip files on the URL classpath to force a timestamp check once */
+void JNICALL
+Java_com_ibm_oti_shared_SharedClassURLClasspathHelperImpl_notifyClasspathChange3(JNIEnv* env, jobject thisObj, jint helperID, jobject classLoaderObj, jobjectArray urlArrayObj, jint urlIndex, jint urlCount, jboolean isOpen)
+{
+#if defined(J9VM_OPT_SHARED_CLASSES)
+	J9VMThread* vmThread = ((J9VMThread*)env);
+	J9JavaVM* vm = vmThread->javaVM;
+	J9ClassPathEntry* cpEntries = NULL;
+	UDATA entryIndex = (UDATA)urlIndex;
+	UDATA entryCount = (UDATA)urlCount;
+	omrthread_monitor_t jclCacheMutex;
```

Comment:
This can be initialized in the declaration.

Reply:
Already updated as suggested.